lib/node_modules/@stdlib/blas/base/sasum/lib/ndarray.js

```diff
@@ -47,14 +52,36 @@ function sasum( N, x, stride, offset ) {
 	var sum;
 	var ix;
 	var i;
+	var m;
 
 	sum = 0.0;
 	if ( N <= 0 ) {
 		return sum;
 	}
 	ix = offset;
+
+	// If the stride is equal to `1`, use unrolled loops...
```

Comment:
@aman-095 You should revert this change. Loop unrolling makes the most sense for simple arithmetic operations. Here, L78 is incorrect and requires `f32()` wrapping for each `+` operation. Better to just use a simple `for` as at L83.

Reply:
Should I also revert the changes for `sasum_ndarray.c` or is it fine.

Reply:
The C implementation is fine, as we are using single-precision arithmetic and don't have the extraneous casting calls.